src/error.rs

```diff
@@ -42,8 +42,8 @@ pub enum SlateDBError {
     #[error("Invalid sst error: {0}")]
     InvalidFlatbuffer(#[from] flatbuffers::InvalidFlatbuffer),
 
-    #[error("Invalid DB state error")]
-    InvalidDBState,
+    #[error("Invalid DB state error: {0}")]
+    InvalidDBState(String),
```

Comment:
This should be reverted / pulled into separate PR ... I needed it for debugging.